sycl/unittests/kernel-and-program/HelperKernelInfo.hpp

```diff
@@ -0,0 +1,29 @@
+//==----------------------- HelperKernelInfo.hpp ---------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/sycl.hpp>
+
+__SYCL_INLINE_NAMESPACE(cl) {
```

Comment:
FYI.
I like this change, but I think we can do even better.
I recently touched unit tests as well and made following changes in https://github.com/intel/llvm/pull/5240.
1. Removed sycl.hpp includes - https://github.com/intel/llvm/pull/5240/commits/04e386940ae915093f622cfe6268c125bab24aeb
2. Moved common code to a helper header - https://github.com/intel/llvm/pull/5240/commits/ba29bbe3157dee3f375ae84dfa90c99be924d7a4